apps-rendering/package.json

```diff
@@ -1,127 +1,128 @@
+	"author": "",
+	"license": "ISC",
+	"prettier": "@guardian/prettier",
+	"dependencies": {
+		"@babel/core": "^7.18.13",
+		"@babel/preset-env": "^7.15.6",
+		"@creditkarma/thrift-server-core": "^1.0.4",
+		"@emotion/cache": "^11.4.0",
+		"@emotion/jest": "^11.10.0",
+		"@emotion/react": "^11.4.1",
+		"@emotion/server": "^11.4.0",
+		"@guardian/apps-rendering-api-models": "3.0.0",
+		"@guardian/atoms-rendering": "^23.7.2",
+		"@guardian/bridget": "^2.0.0",
+		"@guardian/cdk": "^47.3.3",
+		"@guardian/common-rendering": "./../common-rendering",
+		"@guardian/content-api-models": "^17.4.0",
+		"@guardian/content-atom-model": "^3.4.0",
+		"@guardian/discussion-rendering": "^10.1.1",
+		"@guardian/eslint-config-typescript": "^0.7.0",
+		"@guardian/eslint-plugin-source-react-components": "^9.0.0",
+		"@guardian/libs": "^10.0.0",
+		"@guardian/node-riffraff-artifact": "^0.3.2",
+		"@guardian/renditions": "^0.2.0",
+		"@guardian/source-foundations": "^7.0.1",
+		"@guardian/source-react-components": "^9.0.0",
+		"@guardian/source-react-components-development-kitchen": "6.0.2",
+		"@guardian/types": "^9.0.1",
+		"@storybook/addon-knobs": "^6.4.0",
+		"@storybook/addons": "^6.4.0",
+		"@storybook/api": "^6.4.0",
+		"@storybook/builder-webpack5": "^6.4.0",
+		"@storybook/components": "^6.4.0",
+		"@storybook/core-events": "^6.4.0",
+		"@storybook/manager-webpack5": "^6.4.0",
+		"@storybook/react": "^6.4.0",
+		"@storybook/theming": "^6.4.0",
+		"@types/clean-css": "^4.2.5",
+		"@types/compression": "^1.7.2",
+		"@types/express": "^4.17.13",
+		"@types/html-webpack-plugin": "^3.2.6",
+		"@types/jest": "^28.1.6",
+		"@types/jsdom": "^16.2.13",
+		"@types/node": "^14.17.19",
+		"@types/react": "^17.0.39",
+		"@types/react-dom": "^17.0.9",
+		"@types/react-test-renderer": "^17.0.1",
+		"@types/uuid": "^8.3.1",
+		"@types/webpack": "^4.41.31",
+		"@typescript-eslint/eslint-plugin": "^4.33.0",
+		"@typescript-eslint/parser": "^4.33.0",
+		"aws-cdk": "2.39.1",
+		"aws-cdk-lib": "2.39.1",
+		"aws-sdk": "^2.1231.0",
+		"babel-loader": "^9.1.0",
+		"buffer": "^6.0.3",
+		"clean-css": "^5.3.0",
+		"compression": "^1.7.4",
+		"constructs": "10.1.91",
+		"core-js": "^3.24.1",
+		"eslint": "^7.32.0",
+		"eslint-config-prettier": "^8.5.0",
+		"eslint-plugin-eslint-comments": "^3.2.0",
+		"eslint-plugin-import": "^2.26.0",
+		"eslint-plugin-jsx-a11y": "^6.6.1",
+		"eslint-plugin-prettier": "^4.0.0",
+		"eslint-plugin-react": "^7.31.7",
+		"express": "^4.18.1",
+		"html-webpack-plugin": "^5.5.0",
+		"jest": "^28.1.3",
+		"jest-environment-jsdom": "^28.1.3",
+		"jsdom": "^16.7.0",
+		"node-fetch": "^3.2.10",
+		"node-int64": "^0.4.0",
+		"prettier": "^2.6.1",
+		"react": "^17.0.2",
+		"react-dom": "^17.0.2",
+		"react-test-renderer": "^17.0.2",
+		"regenerator-runtime": "^0.13.9",
+		"require-from-string": "^2.0.2",
+		"source-map-support": "^0.5.21",
+		"thrift": "^0.16.0",
+		"ts-jest": "^28.0.7",
+		"ts-loader": "^9.4.1",
+		"ts-node": "^10.8.2",
+		"tslib": "^2.4.0",
```

Comment:
Similar comment to @mxdvl, curious what this is used for?

Reply:
its a peer dep of libs 10